src/ck-ldb/manager.C

```diff
@@ -29,10 +29,10 @@ static void handler(char *bit_map)
     pending_realloc_state = REALLOC_MSG_RECEIVED;
 
     if((CkMyPe() == 0) && (load_balancer_created))
-    LBDatabaseObj()->set_avail_vector(bit_map);
```

Comment:
Is there a reason for this file to exist, rather than just having all this defined with the manager?

Reply:
I'm not sure, but considering that it's it's solely concerned with shrink/expand, I think it should be deferred for now.